.github/workflows/release.yml

```diff
@@ -0,0 +1,66 @@
+# Purpose:
+# Automatically bumps the project's patch version bi-weekly on Fridays.
+#
+# Details:
+# - Triggered at 00:00 UTC every Friday; runs on even weeks of the year.
+# - Sets up the environment by checking out the repo and setting up Node.js.
+# - Bumps patch version using `npm version patch`, then creates a new branch 'release/x.x.x'.
+# - Pushes changes and creates a PR to `main` using GitHub CLI.
+# - Can also be triggered manually via `workflow_dispatch`.
+name: Version Bump
+
+on:
+  workflow_dispatch: # Allow to manually trigger the workflow
+  schedule:
+    - cron: "0 0 * * 5" # At 00:00 UTC every Friday
+
+jobs:
+  version-bump:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Since cronjob syntax doesn't support bi-weekly schedule, we need to check if it's an even week or not
+      - name: Check if it's an even week
+        id: check-week
```

Comment:
Do we need `id`s? I don't see this particular step results being referenced elsewhere.

Reply:
fixed!